docs/building.md

```diff
@@ -268,7 +268,60 @@ Once you've configured your build directory, you can build with
 
 #### Advanced
 LANL Employees - to understand how the project space is built out, see
-https://xcp-gitlab.lanl.gov/eap-oss/parthenon-project
+https://re-git.lanl.gov/eap-oss/parthenon-project
+
+### LANL Snow (CTS-1)
+
+#### Allocate Node
+
+Snow is a LANL CTS-1 system with dual socket Broadwell Intel CPUs. You can log
+in to `sn-fey`. Nodes are allocated using SLURM.
+
+E.g.
+```bash
+$ salloc -N1
+```
+
+#### Set-Up Environment (Optional, but Recommended)
+
+You can import all tools you need to start building with by sourcing the
+project `.bashrc`:
+
+```bash
+$ source /usr/projects/parthenon/parthenon-project/.bashrc
+```
+
+This .bashrc will set the correct `MACHINE_CFG` file in your environment, import
+an architecture-specific set of recent build tools (currently cmake and ninja),
+and set Ninja as the default CMake generator.
+
+#### Build the Code
+If you followed the "Set-Up Environment" section, configuration requires 0
+additional arguments:
+```bash
+$ cmake -S. -Bbuild
+```
+
+If you didn't follow the "Set-Up Environment" section, you need to specify the
+`MACHINE_CFG` file, as well.
+
+```bash
+$ cmake -S. -Bbuild -DMACHINE_CFG=cmake/machinecfg/Snow.cmake
+```
+
+Parthenon is built with the Intel compilers by default on Snow. To build with
```

Comment:
If Intel is the default I also suggest to add additional compiler flags (e.g., to use the simd statements)

Reply:
I think we already are - we're "linking" `OpenMP::OpenMP_CXX` by default.

Reply:
Out of curiosity, would you mind double checking that `-fopenmp-simd` and `-qopenmp-simd`, respectively, are added as compile flags?
I'm not sure linking against OpenMP makes this happen as `openmp-simd` is independent of openmp threading.

Reply:
`-qopenmp` and `-fopenmp` are applied, and the documentation in icpc for `-qopenmp-simd` says this:

```
-qopenmp-simd
          Enables OpenMP* SIMD compilation.  Enabled by default with
          -qopenmp.  Use -qno-openmp-simd to disable.
```

Reply:
Thanks for confirming.